src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -667,18 +674,26 @@ def generate_pvt_model_compositional(value: DescriptionDocument):
     }
 
 
+def load_combined_fluid_description(
+    document: DescriptionDocument,
+) -> Dict[str, case_description.CombinedFluidDescription]:
+    return load_dict_of_instance(document, case_description.CombinedFluidDescription)
+
+
+def load_pvt_model_combined_description(
+    document: DescriptionDocument,
+) -> Dict[str, case_description.PvtModelCombinedDescription]:
+    return load_dict_of_instance(document, case_description.PvtModelCombinedDescription)
+
+
 def load_pvt_models_description(
     document: DescriptionDocument,
 ) -> case_description.PvtModelsDescription:
-    alfacase_to_case_description = {
```

Comment:
Just to clarify. You replaced this code with `load_instance` that already exists? You didn't implement it, right?